apps/api/src/modules/notifications/notifications.service.ts

```diff
@@ -36,8 +36,8 @@ export class NotificationsService {
       throw new BadRequestException(`Channel ${notification.channelType} is not enabled`);
     }
 
-    notification.createdBy = this.configService.getOrThrow<string>('APP_NAME') || 'osmo_notify';
-    notification.updatedBy = this.configService.getOrThrow<string>('APP_NAME') || 'osmo_notify';
+    notification.createdBy = this.configService.getOrThrow<string>('APP_NAME') || 'osmoX';
```

Comment:
Update to OsmoX

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b